motionplan/kinematic_test.go

```diff
@@ -39,7 +39,7 @@ func TestForwardKinematics(t *testing.T) {
 	expect := []float64{300, 0, 360.25, 0, 1, 0, 0}
 	pos, err := ComputePosition(m, &pb.JointPositions{Values: []float64{0, 0, 0, 0, 0}})
 	test.That(t, err, test.ShouldBeNil)
-	actual := poseToSlice(pos)
+	actual := poseToSlice(spatial.PoseToProtobuf(pos))
```

Comment:
This test is ooooooold (hence pose stored as a float array) and should be updated to just test PoseAlmostCoincident

Reply:
modernized the whole thing lol